src/Analyzers/CSharp/CodeFixes/ConvertToInterpolatedString/ConvertToInterpolatedStringCodeFixProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.ConvertToInterpolatedString;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageService;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.ConvertToInterpolatedString;
+
+[ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.ConvertToInterpolatedString), Shared]
+internal partial class ConvertToInterpolatedStringCodeFixProvider : AbstractConvertToInterpolatedStringCodeFixProvider
```

Comment:
likely could be in common layer.  will dpeend if the fix provider itself needs to do something specific for c#.

Reply:
@CyrusNajmabadi I don't know anything about VB, but this doesn't seem too hard to refactor later, so I suppose I can just keep this conversation open and resolve later?